tests/hip/vadd-stream/main.cpp

```diff
@@ -72,14 +72,14 @@ mainworkerthread(hipFunction_t function, hipStream_t stream, bool validate = tru
 
   std::cout << "*********************************************************************************\n";
 
-  std::unique_ptr<float[]> host_a(new float[vector_length]);
-  std::unique_ptr<float[]> host_b(new float[vector_length]);
-  std::unique_ptr<float[]> host_c(new float[vector_length]);
+  std::array<float, vector_length> host_a{};
+  std::array<float, vector_length> host_b{};
+  std::array<float, vector_length> host_c{};
```

Comment:
Array may be problematic depending on size.  This is stack memory.  I think std::vector will be better / safer.